ThunderRequest/RequestController+Callbacks.swift

```diff
@@ -74,6 +79,10 @@ extension RequestController {
         
         NotificationCenter.default.post(name: RequestController.DidReceiveResponseNotificationName, object: nil, userInfo: requestInfo)
         
+        if let urlError = error as? URLError, urlError.code == .notConnectedToInternet {
```

Comment:
Should we be more generic here and have a notification similar to below? `RequestController.DidNetworkErrorNotificationName`? We could just whack the error into the `requestInfo` dictionary and do the checking in the consuming app then?

Reply:
Hmmm that's not a bad point at all. Maybe al `URLError`s? can easily then check for `urlError.code == .notConnectedToInternet`. I'll make that change if you agree? Or have any other suggestions?

Reply:
Updated